deploy.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+cd dest
+
+echo "www.manageiq.org" > CNAME
+
+git config --global user.name "ManageIQ Bot"
+git config --global user.email "noreply@manageiq.org"
+
+git clone --bare --branch master --depth 5 https://miq-bot:${GITHUB_TOKEN}@github.com/ManageIQ/manageiq.github.io .git
```

Comment:
Is there a space before the `.git`?

Reply:
Yes...that is separating the URL from the name of the target dir (which is `.git`)

Reply:
Put another way...this could have also been: `https://.../manageiq.github.io.git .git`

Reply:
Oh, I see, sorry for the confusion.  I was thinking that was supposed to be part of the URL